styles.css

```diff
@@ -0,0 +1,161 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v8.4.14,
+* Autoprefixer: v10.4.7
+* Browsers: last 4 version
+*/
+
+body {
+    margin: 0 auto;
+    font-family: "Poppins", sans-serif;
+    background-image: url(./img/mobBg.png);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position-y: 1.7rem;
+}
```

Comment:
- [ ] Dear @uuukiiiyooo, your design is cool, But as the previous reviewer mentioned we need to have a design that is closely aligned with the figma template, this includes `font, colors, photos, text, margins`. Please see how your design needs `size` and `space` adjustments and address the issue. 
- [ ] Also, Your font size seems to be bigger compared with the figma design, This is also pushing your content outside of the viewport. Please adjust this for a better design. 

### Your design 
![Screenshot 2023-02-17 200835](https://user-images.githubusercontent.com/69077061/219718713-2508cc26-e66c-4ee7-9a9e-086321af5068.png)

### Figma design
![Screenshot 2023-02-17 200924](https://user-images.githubusercontent.com/69077061/219718730-5bbd6281-f796-429f-a6f8-a70c7d83d8f8.png)

Reply:
Hi @Tadesse-Alemayehu! Thanks for your kind feedback, I made the chages to 360 instead of 375, I'll do my best this time!

Reply:
Dear @uuukiiiyooo, Please base your design on `375px` screen size and make sure it looks identical to the figma design. This way the larger screen will also adapt. good luck 🙌